docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<p>I have a stupid question. Does DAMP only produce the top-1 discord? Is there an easy way to get the top-k discords?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
It turns out one can use the following statement to get more than one discord.

see [DAMP_2.0 How To Use.pptx](https://docs.google.com/presentation/d/1_-LGilUJpYRbRZpitw05EgkiOZX52kRd/edit?usp=sharing&ouid=115719825860849832698&rtpof=true&sd=true)
where the authors say:
> The highest peak before the top-1 discord, has a correct value

However, IIUC, this approach does not necessarily give the top-k discord (from left matrix profile). E.g. let's say I have the following approximate left matrix profile vector:

[1, 2, 3, 4, 3.9, 3.8, 3.9, 3.8, 5, 3.7, 4.1]

In this case, the top-1 discord has the discord distance `5 ` (at index `idx`).  Then, we can say that the next top discord in `T[:idx]` (NOT in `T`) is `4`. However, what if I want to consider the whole `T`? 

**Solution:**
I have not thought this through but we might be able to use a sorted vector of length `k` to keep track of top-k discord and do prunning according to the k-th discord distance. I need to:
(1) Check paper 
(2) and/or Check code 
(3) and/or Think about this logic
to see if this can work!

Reply:
> However, what if I want to consider the whole T?

We probably want to be able to do both (i.e., get top-k discords from left MP and get top-k from global MP). What do you think?

Reply:
I am not sure if the current, proposed DAMP can find top-k from global MP. However, we may think later and see how to add such support. (I have some rough idea but haven't implemented it yet).

**Note that my previous comment was regarding left MP.** I THINK the proposed approach for top-k in DAMP does not give the top-k from left matrix profile. Instead, it finds the second-top discord that is located before the top-discords, and find the third-top discord that is located before the second-top discord, and so on.  I THINK THIS is different than finding the top-k from left matrix profile.

Reply:
Related: 
https://sites.google.com/view/discord-aware-matrix-profile/documentation#h.3jzxuzat4jfj

Reply:
Thanks for the suggestion! Will consider it in the new PR#920.